README.md

```diff
@@ -47,34 +47,55 @@ If you're using the Strimzi User Operator, the values for these environment vari
+| `KAFKA_VERSION` | version of the Kafka cluster | `2.8.0` |  |
+| `SARAMA_LOG_ENABLED` | enables the Sarama client logging. | `false` | `saramaLogEnabled` |
+| `VERBOSITY_LOG_LEVEL` | verbosity of the tool logging. Allowed values 0 = INFO, 1 = DEBUG, 2 = TRACE | `0` | `verbosityLogLevel` |
+| `TLS_ENABLED` | if the canary has to use TLS to connect to the Kafka cluster. | `false` |  |
+| `TLS_CA_CERT` | TLS CA certificate, in PEM format, to use to connect to the Kafka cluster. When this parameter is empty (default behaviour) and the TLS connection is enabled, the canary uses the system certificates trust store. When a TLS CA certificate is specified, it is added to the system certificates trust store | empty |  |
+| `TLS_CLIENT_CERT` | TLS client certificate, in PEM format, to use for enabling TLS client authentication against the Kafka cluster. | empty |  |
+| `TLS_CLIENT_KEY` | TLS client private key, in PEM format, to use for enabling TLS client authentication against the Kafka cluster. | empty |  |
+| `TLS_INSECURE_SKIP_VERIFY` | if the underneath Sarama client has to verify the server's certificate chain and host name. | `false` |  |
+| `SASL_MECHANISM` | mechanism to use for SASL authentication against the Kafka cluster. Supported are `PLAIN`, `SCRAM-SHA-256` and `SCRAM-SHA-512`. | empty |  |
+| `SASL_USER` | username for SASL authentication against the Kafka cluster when one of `PLAIN`, `SCRAM-SHA-256` or `SCRAM-SHA-512` is used. | empty |  |
+| `SASL_PASSWORD` | password for SASL authentication against the Kafka cluster when one of `PLAIN`, `SCRAM-SHA-256` or `SCRAM-SHA-512` is used. | empty |  |
+| `CONNECTION_CHECK_INTERVAL_MS` | it defines how often the tool has to check the connection with brokers (in ms). | `120000` |  |
+| `CONNECTION_CHECK_LATENCY_BUCKETS` | buckets of the histogram related to the broker's connection latency metric (in ms). | `100,200,400,800,1600` |  |
+| `STATUS_CHECK_INTERVAL_MS` | it defines how often (in ms) the tool updates internal status information (i.e. percentage of consumed messages) to expose outside on the corresponding HTTP endpoint. | `30000` |  |
+| `STATUS_TIME_WINDOW_MS` | it defines the sliding time window size (in ms) in which status information are sampled. | `300000` |  |
+| `MUTABLE_CONFIG_FILE` | location of an optional external config file that provides configuration at runtime . | empty |  |
+| `MUTABLE_CONFIG_WATCHER_INTERVAL` | interval that mutable config file is examined for changes in content (in ms)  | `30000` |  |
+
+
+## Configuration file
+
+As mentioned above certain aspects of behaviour can be overridden dynamically at runtime from a JSON configuration file.   If a config file reference is
+provided (`MUTABLE_CONFIG_FILE`), that file will be monitored for changes in content and creation/deletion, with any changes being applied dynamically to the
+Canary's runtime state.  In a kubernetes environment this file could be provided by a projected configmap.
```

Comment:
Too many spaces between sentences (+ line-breaks in the middle of sentence seem weird).